src/com/cft/contactmerge/contact/Name.java

```diff
@@ -0,0 +1,92 @@
+package com.cft.contactmerge.contact;
```

Comment:
It looks like this class is very similar to NameProperty.java. In the future, it is better to use "git mv" to rename the file versus creating a new file. That way we can see the changes in the diff and the tracking history we show that the object was renamed.

Reply:
Good to know.